apps/judicial-system/web/src/components/InfoCard/useInfoCardItems.tsx

```
@@ -300,17 +312,42 @@ const useInfoCardItems = () => {
+            className={cn(
+              workingCase.civilClaimants &&
+                workingCase.civilClaimants.length > 1
+                ? styles.renderDivider
+                : undefined,
+              workingCase.civilClaimants &&
+                index === workingCase.civilClaimants.length - 1
+                ? styles.last
+                : undefined,
+            )}
+          >
+            <CivilClaimantInfo
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)